data/flags/man_made/flagpole.json

```diff
@@ -279,6 +279,19 @@
         "subject": "Tasmania",
         "subject:wikidata": "Q34366"
       }
+    },
+      {
+      "displayName": "AU - South Sea Islanders",
+      "locationSet": {"include": ["001"]},
+      "tags": {
+        "country": "AU",
```

Comment:
`country` should is for national flags, not cultural flags. It isn’t necessary to include “AU” in the display name either, unless there’s a distinct flag called “South Sea Islanders” that’s used elsewhere.